api/src/pcapi/sandboxes/scripts/creators/industrial/create_new_caledonia_offerers.py

```diff
@@ -204,6 +207,49 @@ def _create_nc_new_offerer() -> None:
     )
 
 
+def _create_nc_cinema() -> None:
+    address = geography_factories.AddressFactory(
+        street="Rue F. Tisiot",
+        postalCode="98880",
+        city="La Foa",
+        latitude=-20.9361728,
+        longitude=164.4962035,
+        inseeCode="98881",
+        banId=None,
+        timezone="Pacific/Noumea",
+    )
+    offerer = offerers_factories.CaledonianOffererFactory(
+        name="Cinéma De La Foa",
+        street=address.street,
+        postalCode=address.postalCode,
+        city=address.city,
+        siren=siren_utils.rid7_to_siren("1230003"),
+    )
+    venue = offerers_factories.CaledonianVenueFactory(
+        managingOfferer=offerer,
+        pricing_point="self",
+        name=offerer.name,
+        siret=siren_utils.ridet_to_siret("1230001003"),
```

Comment:
Le RIDET doit être cohérent avec le RID7 de la structure parente : 
```suggestion
        siret=siren_utils.ridet_to_siret("1230003003"),
```